bid.proto

```diff
@@ -15,25 +15,23 @@ message Bid {
     JubJubCompressed commitment = 5;
     BlsScalar elegibility_ts = 6;
     BlsScalar expiration_ts = 7;
-    uint64 bid_tree_storage_index = 8;
+    fixed64 bid_tree_storage_index = 8;
```

Comment:
I don't think that's correct. The position/index will be returned with the `BidTransaction` message. But it's not part of the Bid object IMO.

Reply:
I see. In that case I will remove it.